README.rst

```diff
@@ -204,6 +204,13 @@ emails are affected and not the local git commits::
 
   $ git publish --to patches@example.org --signoff
 
+git-publish supports the --dry-run option which is passed through to
+git-send-email to see what would be sent without actually sending anything::
```

Comment:
The --inspect-emails option offers something similar but happens between git-format-patch(1) and git-send-email(1).  This means the CC list may contain sendemail.aliasesFile aliases instead of real email addresses.

Did you consider modifying inspect_menu() to call git-send-email --dry-run so the user can see the final email output?  I think that would make the existing --inspect-emails option more useful.